src/lightning/pytorch/loggers/litlogger.py

```diff
@@ -0,0 +1,414 @@
+            staging_dir: Optional directory where the model will be downloaded.
+            verbose: Whether to show progress bar.
+            version: Optional version string for the model.
+
+        Returns:
+            The loaded model object.
+
+        """
+        import litlogger
+
+        _ = self.experiment
+        return litlogger.get_model(staging_dir, verbose, version)
+
+    @rank_zero_only
+    def get_model_artifact(self, path: str, verbose: bool = False, version: Optional[str] = None) -> str:
+        """Download a model artifact file or directory from cloud storage using litmodels.
+
+        Args:
+            path: Path where the model should be saved locally.
+            verbose: Whether to show progress bar during download.
+            version: Optional version string for the model.
+
+        Returns:
+            str: The local path where the model was saved.
+
+        """
+        import litlogger
+
+        _ = self.experiment
+        return litlogger.get_model_artifact(path, verbose, version)
+
+    @rank_zero_only
+    def log_file(self, path: str) -> None:
+        """Log a file as an artifact to the Lightning platform.
+
+        The file will be logged in the Teamspace drive,
+        under a folder identified by the experiment name.
+
+        Args:
+            path: Path to the file to log.
+
+        Example::
+            logger = LightningLogger(...)
+            logger.log_file('config.yaml')
+
+        """
+        import litlogger
+
+        _ = self.experiment
+        litlogger.log_file(path)
+
+    # ──────────────────────────────────────────────────────────────────────────────
+    # Callback methods
+    # ──────────────────────────────────────────────────────────────────────────────
+
+    def after_save_checkpoint(self, checkpoint_callback: ModelCheckpoint) -> None:
+        """Called after a checkpoint is saved.
+
+        Logs checkpoints as artifacts if enabled.
+
+        """
+        if self._log_model is False:
+            return
+        if checkpoint_callback.save_top_k == -1:
+            self._scan_and_log_checkpoints(checkpoint_callback)
+        else:
+            self._checkpoint_callback = checkpoint_callback
+
+    # ──────────────────────────────────────────────────────────────────────────────
+    # Private methods
+    # ──────────────────────────────────────────────────────────────────────────────
+
+    def _scan_and_log_checkpoints(self, checkpoint_callback: ModelCheckpoint) -> None:
+        """Find new checkpoints from the callback and log them as model artifacts."""
+        checkpoints = _scan_checkpoints(checkpoint_callback, self._logged_model_time)
+
+        for timestamp, path_ckpt, _score, tag in checkpoints:
+            if not self._checkpoint_name:
+                self._checkpoint_name = self.experiment.name
+            # Ensure the version tag is unique by appending a timestamp
+            unique_tag = f"{tag}-{int(datetime.utcnow().timestamp())}"
```

Comment:
```suggestion
            unique_tag = f"{tag}-{int(datetime.now(timezone.utc).timestamp())}"
```


The method `datetime.utcnow()` seems to have been deprecated in Python 3.12. According to the documentation, using `datetime.now(timezone.utc)` is the recommended alternative: https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow